pom.xml

```diff
@@ -1,40 +1,63 @@
-<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+<?xml version="1.0" encoding="UTF-8"?>
+<project
+        xmlns="http://maven.apache.org/POM/4.0.0"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+
     <modelVersion>4.0.0</modelVersion>
+    <prerequisites>
+        <maven>3.0.0</maven>
+    </prerequisites>
+
     <groupId>uk.gov.pay</groupId>
-    <version>0.1-SNAPSHOT</version>
     <artifactId>pay-direct-debit-connector</artifactId>
-
+    <version>0.1-SNAPSHOT</version>
+    <packaging>jar</packaging>
```

Comment:
Do we need this? We are not really packaging dd connector (or any java app) as a jar

Reply:
I thought we are using the generated `jar` here:
- https://github.com/alphagov/pay-direct-debit-connector/blob/f3830c54a32d427df77341ccfb9bd108e3c137e1/Dockerfile#L18
- https://github.com/alphagov/pay-direct-debit-connector/blob/f3830c54a32d427df77341ccfb9bd108e3c137e1/docker-startup.sh#L3
Or you mean something else? Thanks

Reply:
I was just wondering if it was required to put it in the pom, because I think it's the default packaging if not specified. After clarifying you're doing it to make an implicit explicit, i'd say you can leave it there 👍